### ISSUE TITLE
[figma] Cover the :focus-visible states

### ISSUE THREAD

**Issue by oliviertassinari** [mui-org/mui-design-kits#8]
This is similar to https://material.io/design/interaction/states.html#focus.

> The Figma assets account for all aspects of Accessibility, including keyboard focus states. The Material UI Library I purchased from you does not have these.

https://mui.zendesk.com/agent/tickets/1146

## Your Environment 🌎

v5.0.0

**Comment by adrianmanea** [comment]
Agree with having the "focused" state but for which components? [Material 3 Design Kit](https://www.figma.com/community/file/1035203688168086460/Material-3-Design-Kit) covers "focused" and "pressed" but for _some_ components. Their List Item for example only covers Unselected, Hover, Selected.

I'm asking the _which_ question because components might get exponentially larger and I'm not sure if this is the right way to use Figma at this point:
![image](https://user-images.githubusercontent.com/5917515/140286325-fe82dc44-3c01-4f92-9394-67495b00df22.png)

**Comment by oliviertassinari** [comment]
@adrianmanea We have the focus-visible for the TextField, Radios, Slider, etc. already. I would personally add it to the Button and call it a day. I feel that it's the component that lacks coverage the most.

**Comment by adrianmanea** [comment]
One can always manually set the background color to "Focus" which is already defined in the "Action" object.

> I feel that it's the component that lacks coverage the most.

Yes, that's my main focus now, covering all design-related props
![image](https://user-images.githubusercontent.com/5917515/140486531-0a25e677-76cc-42c7-99b8-43b513020c68.png)

**Comment by oliviertassinari** [comment]
Done in v5.2.0
